device/build.gradle

```diff
@@ -8,8 +42,8 @@ android {
         versionCode 3
         versionName "0.3"
 
-        minSdkVersion 20
```

Comment:
Maybe we can add build.gradle file into root directory which will make
it easier and better?